kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -290,6 +290,11 @@ form .input-group {
     background-color: #40775F !important;
 }
 
+.login-btn, btn-block {
+    display: block;
+    width: 100%;
```

Comment:
Also, could you set the lower button to be centrally aligned? I think it is set to float: right currently.